subiquity/models/filesystem.py

```diff
@@ -958,6 +958,19 @@ def size(self):
         return self.volume.size - LUKS_OVERHEAD
 
 
+@fsobj("device")
+class ArbitraryDevice(_Device):
+    ptable = attr.ib(default=None)
```

Comment:
not that it's a new problem but we're overdue for a ptable enum

Reply:
heh yes. not right now though!